HPXMLtoOpenStudio/schematron_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'pathname'
+require 'schematron-nokogiri'
+
+file_dir = File.dirname(__FILE__)
+
+# load the schematron xml
+stron_doc = Nokogiri::XML File.open(File.join(file_dir, 'resources', 'EPvalidator.stron'))  # "/path/to/schema.stron"
+# make a schematron object
+stron = SchematronNokogiri::Schema.new stron_doc
+# load the xml document you wish to validate
+xml_doc = Nokogiri::XML File.open(File.join(File.dirname(file_dir), 'workflow', 'sample_files', 'base.xml'))  # "/path/to/xml_document.xml"
```

Comment:
Seems like we should be looping through all sample files, not just testing the one file.